app/scripts/controllers/eventRegistrations.js

```diff
@@ -72,8 +72,15 @@ angular.module('confRegistrationWebApp')
         }
       });
     });
+
     //turn on visible blocks
     var visibleBlocks = localStorage.getItem('visibleBlocks:' + conference.id);
+    if(typeof visibleBlocks === 'undefined' || visibleBlocks === null){
+      //Initally display (true) or not display (false) columns
+      angular.forEach($scope.blocks, function(value){
+        value.visible = true;
```

Comment:
@hlbraddock I think the intention was to make the email, started, and completed shown by default, not all of the columns, correct? With all of the columns shown it is really hard to find the columns you want and you spend a long time deselecting them (outside of the scope of this PR but it makes me want a show all and a show none button). I don't think you need this block of code.